src/xml.c

```diff
@@ -525,22 +525,8 @@ xml_get_ns_attribute (xmlNodePtr node, const gchar *name, const gchar *namespace
 	return xmlGetNsProp (node, BAD_CAST name, BAD_CAST namespace);
 }
 
-static void
-liferea_xml_errorSAXFunc (void * ctx, const char * msg,...)
-{
```

Comment:
Is this really a duplicate that returns the same errors ?

For the feed given in #867 it returned `internal error: Huge input lookup` but the new error reporting just reports feed discovery failed. This feed is a weird case though, as parsing doesn't fail because of bad xml ...

Reply:
Thanks for the hint. I'll retest with this feed!

Reply:
I'm readding the liferea_xml_errorSAXFunc() callback so we get extra errors on the console. But due to the signature being different from xml_buffer_parse_error() where an errorCtxtPtr is being passed (while liferea_xml_errorSAXFunc() gets a xmlSaxHandlerPtr) it is impossible to pass the error to the GUI.